low-level/files/opt/seagate/sspl/setup/sspl_cleanup.py

```diff
@@ -0,0 +1,111 @@
+        """Reset and cleanup config."""
+        try:
+            if os.path.exists(file_store_config_path):
+                FileUtils.reset_log_files(file_store_config_path, del_file=True)
+            shutil.copyfile("%s/conf/sspl.conf.%s.yaml" % (
+                SSPL_BASE_DIR, product), file_store_config_path)
+            if self.pre_factory:
+                self.cleanup_log_and_config()
+        except OSError as e:
+            logger.error(f"Failed in Cleanup. ERROR: {e}")
+
+    @staticmethod
+    def cleanup_log_and_config():
+        """--pre-factory cleanup : Cleanup logs, config files and
+        undo everything whatever was done in post-install Mini-Provisioner
+        Interface."""
+        Conf.load(SSPL_CONFIG_INDEX, sspl_config_path)
+        sspl_log_file_path = Utility.get_config_value(
+            SSPL_CONFIG_INDEX, "SYSTEM_INFORMATION>sspl_log_file_path")
+        iem_log_file_path = Utility.get_config_value(
+            SSPL_CONFIG_INDEX, "IEMSENSOR>log_file_path")
+
+        # Directories and file which needs to deleted.
+        directories = [
+            f'/var/{PRODUCT_FAMILY}/sspl', f'/var/{PRODUCT_FAMILY}/iem/',
+            f'/var/log/{PRODUCT_FAMILY}/sspl/', f'/var/log/{PRODUCT_FAMILY}/iem/',
+            '/etc/sspl-ll/', f'{PRODUCT_BASE_DIR}/iem/iec_mapping']
+
+        sspl_sudoers_file = '/etc/sudoers.d/sspl'
+        sspl_dbus_policy_rules = '/etc/polkit-1/rules.d'
+        sspl_dbus_policy_conf = '/etc/dbus-1/system.d'
+        sspl_service_file = '/etc/systemd/system/sspl-ll.service'
+        sspl_test_backup = '/etc/sspl_tests.conf.back'
+        sspl_test_file_path = '/etc/sspl_test_gc_url.yaml'
+        sspl_sb_log_file_path = sspl_log_file_path.replace(
+            "/sspl.log", "/sspl_support_bundle.log")
+        manifest_log_file_path = sspl_log_file_path.replace(
+            "/sspl.log", "/manifest.log")
+        # symlinks created during post_install
+        sspl_ll_cli = "/usr/bin/sspl_ll_cli"
+        # Remove SSPL config other config/log files which we have
+        # created during post_install.
+        for filepath in [
+            sspl_ll_cli, sspl_test_backup, sspl_test_file_path,
+            file_store_config_path, global_config_file_path, sspl_log_file_path,
+            iem_log_file_path, sspl_sb_log_file_path, manifest_log_file_path,
+            RSYSLOG_IEM_CONF, RSYSLOG_SSPL_CONF, RSYSLOG_MSB_CONF,
+            IEM_LOGROTATE_CONF, SSPL_LOGROTATE_CONF, MSB_LOGROTATE_CONF,
+            RSYSLOG_SB_CONF, SB_LOGROTATE_CONF, sspl_dbus_policy_conf,
+                sspl_dbus_policy_rules, sspl_sudoers_file, sspl_service_file]:
+            FileUtils.reset_log_files(filepath, del_file=True)
+        # Delete directories which we have created during post_install.
+        for directory in directories:
+            FileUtils.reset_log_files(directory, del_dir=True)
+        logger.info("Deleted config/log files and directories.")
+        # Delete sspl-ll user
```

Comment:
Delete created topic also

Reply:
done.

Reply:
Add new line before `# Delete sspl-ll user`

Reply:
done.

Reply:
I don't see this change

Reply:
which change??
Topic deletion I have added at last. please see the line130.
and also added one new line before L120: # Delete sspl-ll user.

Reply:
Ok. I was checking older commit. I can see now